src/constants.ts

```diff
@@ -1,12 +1,5 @@
 // Chain names and ID's
-export const CHAIN_IDs = {
-  MAINNET: 1,
-  OPTIMISM: 10,
-  POLYGON: 137,
-  BOBA: 288,
-  ZK_SYNC: 324,
-  BASE: 8453,
-  ARBITRUM: 42161,
+export const TESTNET_CHAIN_IDs = {
```

Comment:
Could be nice to split this out further into GOERLI and SEPOLIA - wdyt?

Reply:
Goerli is slowly being deprecated but various chains are still running and I still have a relayer running there, so I'd like to keep support until we can properly support the Sepolia testnet.